app/scripts/metamask-controller.js

```diff
@@ -3405,6 +3406,13 @@ export default class MetamaskController extends EventEmitter {
     }
   }
 
+  // Throttle the call to update phishing lists in order to constrain # of requests
+  /// while maintaining non-blocking updates every hour while users are actively browsing
+  throttledUpdatePhishingLists = throttle(
```

Comment:
@Gudahtt sorry I might have missed something from our convo... but couldn't we just achieve this by checking whether the list is out of date according the phishing controller refresh interval (as we were doing previously) and then calling `updatePhishingLists` if it is but not waiting/blocking on it?

Reply:
... yes, that makes sense too! Then we could rely on the interval set in the controller. Good point.

Reply:
Maybe lets go with that so that the controller just manages its own refresh intervals?

Reply:
Agreed

Reply:
Done [here](https://github.com/MetaMask/metamask-extension/pull/16033/commits/23c58f5d4e00e4c586403e452a050654f7d5f898)